SimplioOfficial/simplio-app
# fix: validate the scanned address vs networkId

### How to test:
1. Get the correct QR code for example from receive page.
2. Go to Send page.
3. Press Scanner Button and scan that code u have from receive.
4. There should not be `Invalid Address` notification anymore, the address should be copied.
